src/drawing/backend.rs

```diff
@@ -39,6 +39,8 @@ pub trait BackendStyle {
     fn stroke_width(&self) -> u32 {
         1
     }
+
+    fn title(&self) -> &str { "" }
```

Comment:
And why there's a title here?

Reply:
It seems the title is not actually used by any of the backend.

Reply:
Ahhh... i have heavy customized Webgl backend, and when dealing with all Shders, Vertexes and Fragments you need to know what kind of lines you are drawing, sometimes you need to pull through additional color to make some other stuff more smooth. So just treat it like helpful stuff. I'm using it heavily.